_posts/wiki/2019-09-03-Undergrad-Guide.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Getting Started as an Undergraduate in the HIRO Group
+description: Answering the questions you didn't know to ask
+permalink: join_us.html
+---
+
+# Contents
+This guide is intended for new undergraduate members with limited exposure to
```

Comment:
Although I'm fine with calling this "undergraduate guide", I'd wager we should limit the use of the "undergraduate", given that this applies generally to "new members with limited exposure to ROS and ..."